_episodes/05-loops-and-functions.md

```diff
@@ -41,7 +41,7 @@ vulture
 hippo
 ```
 
-The line defining the loop must start with `for` and end with a colon, and the
+Here, the line defining the loop start with `for` and end with a colon, and the
```

Comment:
You've changed the formulation of the sentence, but I'm not sure what added value the new formulation provides. If you'd like this accepted, please explain the purpose of this change. You've also introduced a grammatical error ("the line [...] start").